### PR TITLE
Pad next token chooser parameters with empty logits processors

### PR DESCRIPTION
**Pull request by kdamaszk**
# What does this PR do?

Fixes performance issue while padding next token chooser parameters. Some of the logits processor were active by default for dummy requests what causes unnecessary host overhead for higher batch sizes, especially if most of requests were dummy at the moment.

### CODE REVIEW

**Comment by kdamaszk** [comment]
@regisss please review. There was a small perf regression introduced by #140 visible for very high batch sizes.

**Review by regisss — APPROVED** [review]
LGTM!
